react/src/components/Navbar.js

```diff
@@ -1,7 +1,6 @@
+import { Link } from 'react-router-dom';
 import Styles from './Navbar.module.css';
 import logo from '@/icons/desktop/Logo.png';
-import { Link } from 'react-router-dom';
-
 
 function Navbar(){
```

Comment:
`const Navbar = () => {`

Reply:
Co za zagadka... Ilekroć tego nie poprawię, cały czas to `function` wraca. 😩